src/plots/cartesian/layout_defaults.js

```diff
@@ -39,8 +39,9 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
     var yaMayHide = {};
     var xaMustDisplay = {};
     var yaMustDisplay = {};
-    var yaMustForward = {};
-    var yaMayBackward = {};
+    var yaMustNotReverse = {};
```

Comment:
What was the problem with `Forward` and `Backfard` here?
`yaMustForward` may be less confusing that `yaMustNotReverse`.

Reply:
The attribute value of interest here is `reversed`. There's no `autorange: 'forward'`

Reply:
Thanks. Resolved.

Reply:
I made that change when introducing `yaMayScaleanchor` and `yaMayConstrain` (which have now disappeared).  Finding good name/word to evoke the opposite of `Scaleanchor` and `Constrain` was difficult. Instead, I decided to always reuse the same name/word but prepend a `Not` in front. I would argue that this pattern is simpler: you know right away it's the opposite thing. I realize now however that `MustDisplay` could/should be `MustNotHide`.

@archmoj I can revert it if you and @etpinard  really dislike it.